transform/mattermost-analytics/models/intermediate/sales/int_charges_filtered_onprem.sql

```diff
@@ -0,0 +1,50 @@
+{{config({
```

Comment:
We seemingly are using onprem (here) and self-hosted (see https://mattermost.atlassian.net/wiki/spaces/DATAENG/pages/2435874817/Self-serve) interchangeably. Wonder if we should decide and align on one one term to use across.